docs/docsite/rst/user_guide/playbooks_tests.rst

```diff
@@ -59,7 +59,7 @@ To match strings against a substring or a regular expression, use the "match", "
             msg: "matched pattern 4"
           when: url is regex("example.com/\w+/foo")
 
-'match' requires a complete match in the string, while 'search' only requires matching a subset of the string.
+'match' requires a complete match in the string, while 'search' and 'regex' only require matching a subset of the string.
```

Comment:
This isn't exactly true.  The default values of `regex` make it work just like `search`, but it can be configured to perform other matches as well:

```

def regex(value='', pattern='', ignorecase=False, multiline=False, match_type='search'):
    ''' Expose `re` as a boolean filter using the `search` method by default.
        This is likely only useful for `search` and `match` which already
        have their own filters.
    '''
```